### ISSUE TITLE
operator-sdk generate fails to work to work with modules

### ISSUE THREAD

**Issue by nrvnrvn** [operator-framework/operator-sdk#1618]
## Bug Report

**What did you do?**
```
$ operator-sdk generate k8s --verbose
DEBU[0000] Debug logging is set                         
FATA[0000] Project not in $GOPATH  
```
`go.mod` and `go.sum` are present. The project is valid and it can be built manually.

**What did you expect to see?**
Successfully generating k8s zz_*.generated.go files.

**What did you see instead? Under which circumstances?**
A clear and concise description of what you expected to happen (or insert a code snippet).

**Environment**
* operator-sdk version:
operator-sdk version: v0.8.1, commit: 33b3bfe10176f8647f5354516fff29dea42b6342

* go version:
go version go1.12.6 darwin/amd64

* Kubernetes version information:
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.0", GitCommit:"e8462b5b5dc2584fdcd18e6bcfe9f1e4d970a529", GitTreeState:"clean", BuildDate:"2019-06-20T04:49:16Z", GoVersion:"go1.12.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.0", GitCommit:"e8462b5b5dc2584fdcd18e6bcfe9f1e4d970a529", GitTreeState:"clean", BuildDate:"2019-06-19T16:32:14Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}

* Kubernetes cluster kind: 
minikube

* Are you writing your operator in ansible, helm, or go?
go

**Possible Solution**

**Additional context**

**Comment by nrvnrvn** [comment]
there is no such issue on the master branch. Will stick to it for a while...

**Comment by LiliC** [comment]
@nrvnrvn so you are saying this was fixed in master branch but does not work in 0.8.1 release?

We should maybe cherry-pick the fix to the 0.8.2 release? cc @hasbro17 @estroz

**Comment by estroz** [comment]
Yes I'll release `v0.8.2` today or tomorrow.

**Comment by estroz** [comment]
@nrvnrvn are you running `operator-sdk generate openapi` in `$GOPATH/src`?

**Comment by nrvnrvn** [comment]
Right, I’d switched to go modules and that’s where all the fun began )))

**Comment by tengqm** [comment]
Looks like this issue is solved?

**Comment by nrvnrvn** [comment]
solved with the release of v0.9.0
